### DISCUSSION TITLE
可以支持gzip么？

### DISCUSSION THREAD

**Discussion by snice** [openkraken/kraken#1126] [Ideas]
可以支持gzip么？

**Comment by andycall** [comment]
支持gzip做什么？

**Reply by snice** [reply]
减少网络压力呗，比如 https://raw.githubusercontent.com/openkraken/kraken/master/kraken/example/assets/bundle.js.gz

**Comment by cnryb** [comment]
kraken 是一个渲染引擎。

建议使用 flutter 或者原生代码实现你的需求。比如把相关文件缓存到设备上，甚至内存中。
另外，br 的压缩率要比 GZIP 更高。

**Comment by wssgcg1213** [comment]
Http 的 gzip 就可以 https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Encoding
